<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkNoteSet</class>
 <widget class="QWidget" name="WorkNoteSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>五防机配置</string>
   </property>
   <widget class="QCheckBox" name="lockYXCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>五防机闭锁全部遥信</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoMatchCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动匹配检查项</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="allowWfYkCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>允许机五防遥控</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="promptCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>五防机执行提示操作</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="refreshTicketCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>系统启动时刷新操作票</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mnpConfirmCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>遥控前要求模拟屏确认</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="forbidMBEditorCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>禁止模板票编辑</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="execTicketCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>转为执行票直接执行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sectionOperCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>直接装载断面执行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clearTicketCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>执行取消/结束后钥匙清票</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="forbidModifyCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>禁止预演时拉合操作修改</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lockAllCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>系统启动时闭锁所有间隔</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="judgeMulCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>预演时判断多任务冲突</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="directOperaCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>预演结束后直接执行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="promptNetCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>预演时需提示网络状态</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="forbidPreCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>禁止模板票预演</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveStateCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开票后保存状态继续开票</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clickPromptCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启用画面点击提示操作</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveForbidMBCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>保存时屏蔽模板票</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fromMnpCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>从模拟屏开票</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fujianModelCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>福建模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="promptUndownCheck">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>提示操作不下传</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="printForbidWMCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打印操作票屏蔽网门</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>操作票格式</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>每页</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>31</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>行</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>59</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(&lt;=4个)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>执行窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>结束语设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
